idb/mocks/IndexerDb.go

```diff
@@ -5,12 +5,15 @@ package mocks
 import (
 	context "context"
 
+	bookkeeping "github.com/algorand/go-algorand/data/bookkeeping"
+
 	generated "github.com/algorand/indexer/api/generated/v2"
+
 	idb "github.com/algorand/indexer/idb"
 
 	mock "github.com/stretchr/testify/mock"
 
-	types "github.com/algorand/indexer/types"
+	transactions "github.com/algorand/go-algorand/data/transactions"
```

Comment:
nit: what's going on here lol. 4 groups and no need to rename most of them.

Reply:
this code is autogenerated, so can't take the blame :)